### PR TITLE
pythonPackages.django-csp: init at 3.5

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Motivation for this change

Just adding a missing package.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/django-csp-nonce/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchPypi, buildPythonPackage, django }:
+
+buildPythonPackage rec {
+  pname = "django_csp";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/django-csp-nonce/default.nix]:
```suggestion
  pname = "django-csp";
```

We'd like to normalize `pname` in this manner.

You'll need to change the `pname` in `fetchPypi` or else the fetch will break.

**Comment by mmahut** [comment]
@MrMebelMan do you mind updating the name please?

File: pkgs/development/python-modules/django-csp-nonce/default.nix (hunk 1 above)
Comment (MrMebelMan) [reply pkgs/development/python-modules/django-csp-nonce/default.nix]: sure!

**Comment by MrMebelMan** [comment]
> @MrMebelMan do you mind updating the name please?

done & rebased master

**Comment by worldofpeace** [comment]
Why is it `django-csp-nonce` and not `django-csp`?

**Comment by MrMebelMan** [comment]
> Why is it `django-csp-nonce` and not `django-csp` in all-packages.nix?

oh... I think i've messed it up when packaging both `django-csp` and `django-csp-nonce`.

Changed to django-csp.

**Comment by worldofpeace** [comment]
Thanks @MrMebelMan :sparkles:

**Pull request merged**